CHANGES.md

```diff
@@ -5,8 +5,11 @@
 - Moved all conditions for the calculation of the exposure to be inside batExposure
 - Made all returned values of batExposure be float values (both the area exposure and the cos(theta))
 - Modified simbadlocation to be able to access table RA/Dec values based on lowercase or uppercase headers, which changes with different versions of astroquery
+<<<<<<< HEAD
```

Comment:
This indicates that git had a conflict.  As you clear the conflict, you should remove the conflict markers.